projects/packages/stats/src/class-wpcom-stats.php

```diff
@@ -203,13 +203,18 @@ public function get_file_downloads( $args = array() ) {
 	 * Get a post's views.
 	 *
 	 * @link https://developer.wordpress.com/docs/api/1.1/get/sites/%24site/stats/post/%24post_id/
-	 * @param int   $post_id The video's ID.
-	 * @param array $args    Optional query parameters.
+	 * @param int   $post_id        The post's ID.
+	 * @param array $args           Optional query parameters.
+	 * @param bool  $cache_in_meta  Optional should cache in post meta.
 	 * @return array|WP_Error
 	 */
-	public function get_post_views( $post_id, $args = array() ) {
+	public function get_post_views( $post_id, $args = array(), $cache_in_meta = false ) {
 		$this->resource = sprintf( 'post/%d', $post_id );
 
+		if ( $cache_in_meta ) {
+			$this->fetch_post_stats( $args, $post_id );
```

Comment:
```suggestion
			return $this->fetch_post_stats( $args, $post_id );
```

I believe we should return the result here 😄

Reply:
Whoops, great spot! :)